include/boost/numeric/ublas/tensor/expression.hpp

```diff
@@ -61,12 +67,21 @@ struct binary_tensor_expression
 
     using size_type = typename tensor_type::size_type;
 
-    explicit binary_tensor_expression(expression_type_left  const& l, expression_type_right const& r, binary_operation o)
+    explicit constexpr binary_tensor_expression(expression_type_left  const& l, expression_type_right const& r, binary_operation o)
       : el(l) , er(r) , op(o) {}
     binary_tensor_expression() = delete;
     binary_tensor_expression(const binary_tensor_expression& l) = delete;
-    binary_tensor_expression(binary_tensor_expression&& l)
+    constexpr binary_tensor_expression(binary_tensor_expression&& l) noexcept
       : el(l.el), er(l.er), op(l.op) {}
+    constexpr binary_tensor_expression& operator=(binary_tensor_expression&& l) noexcept{
```

Comment:
Why making expressions assignable? Was it `clang-tidy` warning that said missing custom assignment operator?